lib/IHP/static/helpers.js

```diff
@@ -42,20 +42,20 @@ function initTime() {
 }
 
 function initDelete() {
-    document.addEventListener('click', handleClick);
+    document.querySelectorAll(".js-delete").forEach(e => e.addEventListener('click', handleClick));
```

Comment:
Could we use boring JS here? :) This way these helpers support even older browsers. (Only in the JS files used by the dev tools, we can use more modern JS)

```suggestion
    document.querySelectorAll(".js-delete").forEach(function (e) { e.addEventListener('click', handleClick) });
```

Reply:
Makes sense!